command/base/command.go

```diff
@@ -198,6 +211,10 @@ func printTitle(w io.Writer, s string) {
 func printFlag(w io.Writer, f *flag.Flag) {
 	example, _ := flag.UnquoteUsage(f)
 	if example != "" {
+		// Change 'value' to 'string' for consistency
+		if example == "value" {
+			example = "string"
+		}
```

Comment:
This feels a little strange; having common flags show up as `-http-addr=<value>` looks weird, though